config/zones.json

```diff
@@ -1491,6 +1491,34 @@
     },
     "timezone": "Europe/Paris"
   },
+  "FR-COR": {
+    "bounding_box": [
+      [
+        8.361347,
+        41.315625
+      ],
+      [
+        9.904660,
+        43.047539
+      ]
+    ],
+    "capacity": {
+      "biomass": 2.295,
+      "hydro": 223.266,
+      "solar": 145.123,
+	  "wind": 18,
```

Comment:
indentation issue here

Reply:
I'll fix these indentation issues in a future commit :)